### PR TITLE
WIP: Docker resource docs first commit

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### CODE REVIEW

File: docs/resources/docker.md.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@ platform: linux
 
 # docker
 
-Use the `docker` InSpec audit resource to test configuration data for docker daemon. It is a very comprehensive resource. Please have a look at [docker_container](docker_container) and [docker_image](docker_image), too.
+Use the `docker` InSpec audit resource to test configuration data for the docker daemon. It is a very comprehensive resource. See also: [docker_container](docker_container) and [docker_image](docker_image), too.
```
Comment (jerryaldrichiii) [inline docs/resources/docker.md.erb]: Could we change this to `Docker daemon`?

File: docs/resources/docker.md.erb [hunk 2]
```diff
@@ -25,8 +25,8 @@ or:
 
 where
 
-* `.where()` may specify a specific item and value, to which the matchers are compared
-* `commands`, `ids`, `images`, `labels`, `local_volumes`, `mounts`, `names`, `networks`, `ports`, `sizes`  and `'status'` are valid matchers for `containers`
+* `.where()` may specify a specific item and value, to which the resource parameters are compared
+* `commands`, `ids`, `images`, `labels`, `local_volumes`, `mounts`, `names`, `networks`, `ports`, `sizes`  and `'status'` are valid parameters for `containers`
```
Comment (jerryaldrichiii) [inline docs/resources/docker.md.erb]: Could we remove one of the spaces after `sizes` there are two.

File: docs/resources/docker.md.erb [hunk 3]
```diff
@@ -90,7 +89,9 @@ The following examples show how to use this InSpec audit resource.
       its(%w(Config Healthcheck)) { should_not eq nil }
     end
 
-### Run the DevSec docker baseline  profile
+<br>
+
+## How to run the DevSec docker baseline profile
```
Comment (jerryaldrichiii) [inline docs/resources/docker.md.erb]: Should we capitalize `docker`?

File: docs/resources/docker.md.erb [hunk 4]
```diff
@@ -124,6 +129,13 @@ For a full list of available matchers, please visit our [matchers page](https://
       its('labels') { should include 'License=GPLv2,Vendor=CentOS' }
     end
 
+### object('id')
+
+`object` returns low-level information about docker objects. It is calling [docker inspect](https://docs.docker.com/engine/reference/commandline/info/) under the hood.
```
Comment (jerryaldrichiii) [inline docs/resources/docker.md.erb]: Capitalize `docker objects`?

File: docs/resources/docker.md.erb [hunk 5]
```diff
@@ -145,20 +165,55 @@ For a full list of available matchers, please visit our [matchers page](https://
...
+### tag
+
+    describe docker_container(name: 'an-echo-server') do
+      its('tag') { should eq 'latest' }
     end
+
+### ports
+
+    describe docker_container(name: 'an-echo-server') do
+      its('ports') { should eq "0.0.0.0:1234->1234/tcp" }
```
Comment (jerryaldrichiii) [inline docs/resources/docker.md.erb:205]: Is that really the output of `ports`?...yikes...

**Review by jerryaldrichiii — DISMISSED** [review]
Great work, I've provided some comments. Let me know what you think.

File: docs/resources/docker.md.erb (hunk 1 above)
Comment (kagarmoe) [reply docs/resources/docker.md.erb]: done

File: docs/resources/docker.md.erb (hunk 2 above)
Comment (kagarmoe) [reply docs/resources/docker.md.erb]: done

File: docs/resources/docker.md.erb (hunk 3 above)
Comment (kagarmoe) [reply docs/resources/docker.md.erb]: done

File: docs/resources/docker.md.erb (hunk 4 above)
Comment (kagarmoe) [reply docs/resources/docker.md.erb]: done

File: docs/resources/docker.md.erb (hunk 5 above)
Comment (kagarmoe) [reply docs/resources/docker.md.erb:205]: Yesssss.....

**Review by jerryaldrichiii — APPROVED** [review]
Looks good to me!

**Review by jquick — APPROVED** [review]
Thanks @kagarmoe !

**Pull request merged**